app/views/documents/_timeline_entry.html.erb

```diff
@@ -0,0 +1,61 @@
+<% if entry.whitehall_migration? %>
+  <h4 class="govuk-heading-s govuk-!-margin-bottom-1">
+    <%= t "documents.history.entry_types.whitehall_migration.#{entry.details.entry_type}" %>
+  </h4>
+
+  <div class="app-timeline-entry__dateline">
+    <%= format_date_at_time(entry.created_at) %>
+    <% if entry.created_by %>
+    <%= t "documents.history.by" %> <%= name_or_fallback(entry.created_by)%>
```

Comment:
We can probably just do `entry.created_by.name` rather than needing the name_or_fallback helper as we know it's not going to be a fallback.